server/app/services/mail-service.js

```diff
@@ -184,11 +185,57 @@ async function sendWelcomeMail(email) {
     await transporter.sendMail(mailOptions);
 }
 
+/**
+ * Send a contact form mail
+ *
+ * @param formName {String} french contact form name
+ * @param emails {String[]} recipient email address
+ * @param value {Object} form value
+ * @returns {Promise<void>} Nothing
+ */
+async function sendContactForm(formName, emails, values) {
+
+    const mail = await readFile(path.resolve(__dirname, '../../resources/emails', 'contact.html'), 'utf8');
+
+    for (const email of emails) {
+        const currentMail = mail.replace(REGEX_TOKEN, (matches, replaceToken) => {
+            switch (replaceToken) {
+                case 'MAIL_WEBSITE':
+                    return WEB_CONFIG.publicURL;
+                case 'MAIL_USERNAME':
+                    return email;
+                case 'FORM_NAME':
+                    return formName;
+                case 'FORM_VALUES': {
+                    let html = '<p><ul>';
+                    Object.keys(values).forEach(value => {
+                        html += `<li><b>${TRANSLATION[value] ? TRANSLATION[value].french : value}</b>: ${values[value]}</li>`;
+                    });
+                    html += '</ul></p>';
+                    return html;
+                }
+            }
+        });
+
+        const transporter = nodemailer.createTransport(CONFIG);
+        // Setup email data with unicode symbols
+        const mailOptions = {
+            from: CONFIG.auth.user,
+            to: email,
+            subject: `[K-App] Nouveau formulaire de contact pour un ${formName}`,
+            html: currentMail,
+        };
+
+        await transporter.sendMail(mailOptions);
```

Comment:
I think we don't need to send a custom mail for each email...
We just have to remove the `MAIL_USERNAME`.

For the form values, I don't really know, If we want to add translations, we should use an `i18n` package to do so. Else, we should just have a key-value map (for now directly in the function).

An i18n package would be more work, but will be nice for errors (we know the client language)... 

We just keep french mails for now

Reply:
Ok let's keep french email for now!